client/landing/stepper/declarative-flow/internals/steps-repository/design-setup/unified-design-picker.tsx

```diff
@@ -448,12 +449,19 @@ const UnifiedDesignPickerStep: Step = ( { navigation, flow, stepName } ) => {
 		} else {
 			plan = isEligibleForProPlan && isEnabled( 'plans/pro-plan' ) ? 'pro' : 'premium';
 		}
+
+		// When the user is done with checkout, send them back to the current url
+		// If the theme is externally managed, send them to the marketplace thank you page
+		const destination = selectedDesign?.is_externally_managed
+			? addQueryArgs( `/marketplace/thank-you/${ siteSlug }`, {
+					themes: selectedDesign?.slug,
+			  } )
+			: window.location.href.replace( window.location.origin, '' );
```

Comment:
Absolute minor comment, not even having a level of nitpick 😅. Could a line be added below to separate this piece of code from the call to `goToCheckout`? I initially thought it was incorrectly indented.